lib/web_ui/test/engine/pointer_binding_test.dart

```diff
@@ -1330,6 +1334,7 @@ void testMain() {
           packets[4].data[0].signalKind, equals(ui.PointerSignalKind.scroll));
       expect(
           packets[4].data[0].kind, equals(ui.PointerDeviceKind.trackpad));
+      expect(packets[4].data[0].pointerIdentifier, equals(-2));
```

Comment:
Should be device?

Reply:
Oh 🤦‍♀️  - thanks. Still recovering from jet lag.